Move button functionality

mgray35 opened the pull request:
Move buttons work!

I had to do it the ugly/dirty way because I understand nothing about json or ajax. I only got this to work because I noticed that the "colTo" variable in the moveButton function _doesn't actually do anything_. That being said, removing it broke something (not sure what on account of the lack of json/ajax understanding). So I basically left the function as-is, assigning a dummy value to colTo, and just made 4 different versions of it.

If anyone can figure out how to do this more elegantly, be my guest. For now, I'm just happy they work the way we've wanted them to since sprint 2.

osugraytr commented:
Hell yeah!! Good job!

osugraytr commented:
I was looking at the code and it seems like the move buttons will only move cards to column 0

Code snip:
![image](https://user-images.githubusercontent.com/29607247/33052302-0f45b3e0-ce23-11e7-9764-6e3a91eaf62f.png)

Could you explain why this is the way it is plz? Then I'll merge!:)

kevinkoos commented on src/main/java/views/AcesUp/AcesUp.flt.html:
```diff
@@ -245,6 +382,48 @@ <h1>Aces Up</h1>
   dataType:"json",
 });
 });
+$("#move1").click(function(){
```
Instead of making a listener function for each individual button, make all the move buttons have the same class but different id corresponding to their column. Then use `var id = $(this).attr('id')` to get the individual buttons column. The way cards are placed into their table cells works in a similar way.

osugraytr replied:
Thanks!

mgray35 commented:
@osugraytr I explained that in the PR description. The "colTo" variable doesn't do anything. The java logic will always move the "colFrom" card into the first empty column. I set colTo to be 0 just as a dummy value.

mgray35 commented:
Okay, I cleaned up the js move function and also removed the colTo variable. It was easier than I thought, I just had to take a closer look at how it works.

osugraytr commented:
Thanks for explaining this!! Merging now
